pkgs/development/python-modules/scikit-misc/default.nix

```diff
@@ -0,0 +1,39 @@
+{ buildPythonPackage
+, cython
+, fetchFromGitHub
+, gfortran
+, lib
+, numpy
+, pytest-cov
+, pytestCheckHook
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-misc";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "has2k1";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1h8lxyazvq3b2c8blj131zn8wkbzjb8ynpr2a7yaanblfsbzdv92";
+  };
+
+  nativeBuildInputs = [ cython gfortran numpy ];
+
+  # Testing packages that involve cython is a little tricky. We need to point
+  # pytest to the output directory instead of the source directory. See
+  #   * https://github.com/NixOS/nixpkgs/pull/137038
+  #   * https://discourse.nixos.org/t/cant-import-cythonized-modules-at-checkphase/14207
+  # for more information.
+  checkInputs = [ pytest-cov pytestCheckHook ];
```

Comment:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/137464/commits/2a9a4a57c5b5a090bc72809d1db482c227498e31.